recipes/tracktable/meta.yaml

```diff
@@ -0,0 +1,108 @@
+# https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=meta.yaml
```

Comment:
Please remove all the comments that are not relevant for the final recipe anymore.

Reply:
Extraneous comments are removed.